infra/filestorage.yaml

```diff
@@ -79,10 +79,23 @@ Resources:
               - - 'arn:aws:s3:::'
                 - !Ref bucket
                 - /*
-            Principal: '*'
+            NotPrincipal:
+              AWS: 'arn:aws:iam::405662711265:role/AmazonECSTaskS3BucketRole'
```

Comment:
Thoughts: this makes me realize our staging roles have the same permission as prod roles, although they not restricted by resource related permissions so might be okay.

Reply:
Ah, so we could reach the prod bucket from staging?  That would be worth fixing, haha.  :)  If that's true, we should fix it - if you think it might be true, can you make a github issue?  I had thought this role was restricted to the bucket in the same stack, but I don't know if that's right.

Reply:
I will follow up on this. Merging this PR.